src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -853,6 +861,10 @@ def train(self, training_params: dict = dict()):  # noqa: C901
         else:
             raise UnsupportedOptimizerFormat()
 
+        # VERIFY GRADIENT CLIPPING VALUE
+        if self.training_params.clip_grad_norm is not None and self.training_params.clip_grad_norm <= 0:
+            raise Exception('Params', 'Invalid clip_grad_norm')
```

Comment:
Let's raise type error here instead of exception.

Reply:
fixed

Reply:
fixed